### DISCUSSION TITLE
Differences in PodSecurityPolicy and SecurityContextConstraints config

### DISCUSSION THREAD

**Discussion by sersanherr** [grafana/mimir#3850] [Help and support]
Hello everyone,

We are in the process of installing mimir-distributed in Openshift. We were reviewing the SecurityContextConstraint template here:
https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/templates/securitycontextconstraints.yaml

and found some differences with the PodSecurityPolicy used in normal non-ocp k8s (until is deprecated at least):
https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/templates/podsecuritypolicy.yaml

The SCC for OCP seems to use less restrictive rules than the PSP for non-OCP.

Could someone please explain the reasoning behind these differences in configuration?

Thank you in advance!

**Comment by dimitarvdimitrov** [comment]
hi @sersanherr, we don't have much operational experience with OpenShift at Grafana Labs so IIRC the SCC was created before we picked up creating a decent PSP. There shouldn't be a reason for these differences. 

If you're willing to try out a more restrictive SCC in your own clusters, I'll be happy to incorporate the changes into mimir-distributed. Otherwise, I can try taking a look with a local OpenShift cluster in the next couple of weeks.
